### DISCUSSION TITLE
Video downloads complete immediately with zero-byte output file

### DISCUSSION THREAD

**Discussion by Milburne** [FreeTubeApp/FreeTube#6355] [General]
This began in the last few days, my last successful download was on the 5th of this month. 

I am selecting the 360p video with audio option and get the "starting" and "finished" messages immediately. Correctly named output file is produced but contains no data.

This is using the Freetube 0.22 amd deb installation on Linux Mint 22 Cinnamon. Application and system have both been restarted with no improvement. All other functions seem normal.

**Comment by absidue** [comment]
Duplicate of #6328
